src/inject/dynamic-theme/watch.ts

```diff
@@ -179,9 +179,11 @@ export function watchForStyleChanges(update: (styles: ChangedStyles) => void) {
             return;
         }
         const shadowObserver = new MutationObserver(handleMutations);
-        shadowObserver.observe(node.shadowRoot, mutationObserverOptions);
-        shadowObservers.add(shadowObserver);
-        nodesShadowObservers.set(node, shadowObserver);
+        if (shadowObserver.takeRecords().length != 0) { // Don't handle empty Shadowobservers
```

Comment:
It looks like it will always return 0, since the observer is not observing anything.

Reply:
![](https://i.imgur.com/TjOWFln.png) It's observing a empty {}